package/MDAnalysis/core/topologyattrs.py

```diff
@@ -31,6 +31,8 @@
 from .topologyobjects import TopologyGroup
 from . import selection
 from . import flags
+from .groups import (GroupBase, Atom, Residue, Segment,
```

Comment:
This isn't a problem yet, but I did have in mind that by using `Level` object (that are dependency free) we can avoid circular imports everywhere, and they nicely hash against human readable things like `'atom'`